src/plugins/intel_cpu/src/nodes/mkldnn_fake_quantize_node.cpp

```diff
@@ -1639,6 +1639,36 @@ void MKLDNNFakeQuantizeNode::executeQuantization(const std::unique_ptr<jit_uni_q
             arg.block_size = (size_t) blk_size;
             arg.work_amount = (size_t)H;
 
+            (*pKernel)(&arg);
+        });
+    } else if (jqp.is_planar && srcDims.size() > 2) {
+        const int batch_size = 256;
```

Comment:
minor: usually "batch" name is used to determine 0th dimension (for example, number of images). So here this name is little bit confusing. I would propose to rename like "spatial_batch_size" or "spatial_block_size".